example/ace_check/FindAce.cmake

```diff
@@ -43,7 +43,6 @@ SET(ACE_LINK_FLAGS "${ACE_LINK_FLAGS}" CACHE INTERNAL "ace link flags")
 
 FIND_PATH(ACE_INCLUDE_DIR ace/ACE.h ${CMAKE_SOURCE_DIR}/../ACE_wrappers/ ${CMAKE_SOURCE_DIR}/../../ACE_wrappers/ ${CMAKE_SOURCE_DIR}/../../../ACE_wrappers/ /usr/include /usr/local/include $ENV{ACE_ROOT} DOC "directory containing ace/*.h for ACE library")
 
-# in YARP1, config was in another directory
```

Comment:
I think this comment was related to the 2 following lines...
Anyway this file FindAce.cmake looks very obsolete, the one from YCM should be used instead...